### ISSUE TITLE
Code Structure and understanding requirements

### ISSUE THREAD

**Issue by Anindyadeep** [keras-team/keras#18446]
This is an awesome initiative first of all, Unifiying different AI backends under one standard abstraction layer. I just cloned the repo, I see the current structure of the code. The current structure is with the existing structure of keras, we now have a backend. The backend has similar kind of functions and classes implemented to maintain the extincibility. 

However, some parts of the code requires more contributions and I am not sure which parts of the code are completed and which are not. Yes, I do agree this project is in a very nascent stage. Being an OSS enthusiast just starting out to put contributions, I am highly intersted to know the following things:

1. what are your plans regrading existing code structure? Will there be any major changes or refactor or will follow the existing standards
2. What are the set of functions required for all the backend? For example: Inside `backend/jax/image` and `backend/pytorch/image` both have just one function called `resize`. I am not sure that `resize` is the ONLY function we might be requiring inside `/image`. So it would be awesome to know the requirements (in terms of what are the functions/classes needed to be defined inside the module of interest.)

Please let me know and would be happy to answer contribute here.

**Comment by sachinprasadhs** [comment]
Hi,

Thank you for interest in contributing to the Keras repo, since Keras 3 is rolled out to public, now the team is focused on adding the new features and fixing the bugs.

You can always look for labels contribution welcome to start contributing https://github.com/keras-team/keras/issues?q=is%3Aopen+is%3Aissue+label%3A%22stat%3Acontributions+welcome%22

We encourage community to contribute to the repository. 
You are free to work on any open issues.

**Comment by Anindyadeep** [comment]
Awesome, thanks

**Comment by sachinprasadhs** [comment]
Feel free to close the issue.
